gateway/src/apicast/policy/token_introspection/token_introspection.lua

```diff
@@ -16,20 +17,23 @@ local new = _M.new
 local noop = function() end
 local noop_cache = { get = noop, set = noop }
 
+local function create_credential(client_id, client_secret)
+  return 'Basic ' .. ngx.encode_base64(table.concat({ client_id, client_secret }, ':'))
+end
+
 function _M.new(config)
   local self = new(config)
   self.config = config or {}
+  self.auth_type = config.auth_type or {}
```

Comment:
I think the default value for `auth_type` is "client_id+client_secret", right?

Reply:
Fixed.